### ISSUE TITLE
DRC handling is not thread-safe

### ISSUE THREAD

**Issue by benjamin-weiss** [ittiam-systems/libmpegh#92]
The function `impd_drc_read_uni_drc_gain()` uses static variable `pkt_loss_frm_cnt` to count the number of lost packets and disable DRC if it reaches a certain time. Since it is still a global variable, several decoding threads sharing the same memory space may overwrite that value and cause DRC being disabled for another thread without a reason (or prevent it from being disabled with the counter being reset).

P.S. This seems to be the only instance where a static variable is used in the code and probably it should belong to a decoder context instead.

**Comment by SakethSathuvalli** [comment]
Yes, `pkg_loss_frm_cnt` should have been a decoder context variable.

In the present implementation there will not be a scenario where the value can be anything other than 0. We will clean this part of the code.

Thanks!

**Comment by SakethSathuvalli** [comment]
Changes available on latest main.
